package.json

```diff
@@ -64,7 +64,7 @@
     "imagemin": "imagemin src/images/*.* --out-dir=dist/images/",
     "lint": "run-s lint:js lint:scss lint:php",
     "lint:js": "./node_modules/.bin/eslint src/js || true",
-    "lint:php": "./vendor/bin/phpcs -p -s -n . --standard=WebDevStudios --extensions=php --ignore='node_modules/*,vendor/*,./dist/*,./acf-json/*' || true || exit",
+    "lint:php": "./vendor/bin/phpcs -p -s -n . --standard=.phpcs.xml.dist --extensions=php --ignore='node_modules/*,vendor/*,./dist/*,./acf-json/*' || true || exit",
```

Comment:
Could there be a way to have this run `composer run lint:php` instead, so you have one command to maintain?